.github/workflows/test.yml

```diff
@@ -27,15 +27,14 @@ jobs:
       matrix:
         os: ["macos-latest", "ubuntu-latest", "windows-latest"]
         python-version: ["3.11", "3.12"]
-        parallel: ["", "--test-parallel"]
         storage-layout: ["solidity", "generic"]
         cache-solver: ["", "--cache-solver"]
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
         with:
-          submodules: false
+          submodules: recursive
```

Comment:
submodules are checked out during pytest anyway

Reply:
IIRC it is faster to let foundry do a sparse checkout of the missing libraries rather than having the action do a git checkout of the submodules

Reply:
oh i see. i can revert this